src/index.js

```diff
@@ -45,7 +34,59 @@ server.on('after', (req, res, route, err) => {
   }
 });
 
-const startAPI = async () => {
+let qtumProcess;
```

Comment:
move this variable inside func startQtumProcess

Reply:
`qtumProcess` is used in separate method `exit()`

Reply:
lets try remove `qtumProcess.kill();` from exit(). The reason is that exit of parent process should kill childprocess. `qtumProcess = None` doesnt mean anything, variable reassignment can't guarantee the exit of the process.

Reply:
sounds good